Tests/YamsTests/PerformanceTests.swift

```diff
@@ -58,7 +58,14 @@ class PerformanceTests: XCTestCase {
     ]
 
     func loadYAML() throws -> String {
-        let data = try Data(contentsOf: URL(fileURLWithPath: filename))
+        let data: Data
+        if let buildWorkspace: String = ProcessInfo.processInfo.environment["BUILD_WORKSPACE_DIRECTORY"] {
+            FileManager.default.changeCurrentDirectoryPath(buildWorkspace)
```

Comment:
It’s surprising that changing the working directory would influence the ability to read data from an absolute file path. It also would be preferable to consolidate this path construction:

```swift
private let fixturesDirectory = {
    if let buildWorkspace = ProcessInfo.processInfo.environment["BUILD_WORKSPACE_DIRECTORY"] {
        return "\(buildWorkspace)/Tests/YamsTests/Fixtures/"
    }
    return URL(fileURLWithPath: #file).deletingLastPathComponent().path + "/Fixtures/"
}()
```

Then `fixturesDirectory` is accurate if we want to use it elsewhere in the future and the changes to this test can be reverted (assuming the `changeCurrentDirectoryPath` isn’t actually needed).

Reply:
Fixing now! turns out the `FileManager.default.changeCurrentDirectoryPath` was not needed